Gemfile

```
@@ -3,15 +3,15 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 
 ruby '~> 2.6.5'
 
-gem 'rails', '~> 6.0', '>= 6.0.3.4'
+gem 'rails', '~> 6.1.2'
 
 gem 'activerecord-import', '>= 1.0.2'
 gem 'aws-sdk', require: false
 gem 'bloomfilter-rb'
 gem 'figaro'
 gem 'health_check', '>= 3.0.0'
 gem 'identity-hostdata', github: '18F/identity-hostdata', branch: 'master'
-gem 'lograge'
+gem 'identity-logging', github: '18F/identity-logging', branch: 'v0.1.0'
```

Curious: Should this be... ?

```suggestion
gem 'identity-logging', github: '18F/identity-logging', tag: 'v0.1.0'
```

---

Yes 🤦  thanks